app/models/in_progress_form.rb

```diff
@@ -9,6 +9,14 @@ def cast(value)
     alias serialize cast
   end
 
+  RETURN_URL_SQL = "CAST(metadata -> 'return_url' AS text)"
+  scope :has_attempted_submit, -> { where("(metadata -> 'submission' ->> 'has_attempted_submit')::boolean") }
+  scope :has_errors,           -> { where("(metadata -> 'submission' -> 'errors') IS NOT NULL") }
+  scope :has_no_errors,           -> { where.not("(metadata -> 'submission' -> 'errors') IS NOT NULL") }
+  scope :has_error_message, -> { where("(metadata -> 'submission' -> 'error_message')::text !='false'") }
+  # the double quotes in return_url are part of the value
+  scope :return_url, ->(url) { where(%( #{RETURN_URL_SQL} = ? ), '"' + url + '"') }
```

Comment:
```suggestion
  RETURN_URL_SQL = "CAST(metadata -> 'return_url' AS text)"
  scope :has_attempted_submit, -> { where("(metadata -> 'submission' ->> 'has_attempted_submit')::boolean") }
  scope :has_errors,           -> { where("(metadata -> 'submission' -> 'errors') IS NOT NULL") }
  scope :has_no_errors,        -> { where.not("(metadata -> 'submission' -> 'errors') IS NOT NULL") }
  scope :has_error_message,    -> { where("(metadata -> 'submission' -> 'error_message')::text !='false'") }
  # the double quotes in return_url are part of the value
  scope :return_url, ->(url) { where(%( #{RETURN_URL_SQL} = ? ), '"' + url + '"') }
```

Just aligning the params